src/components/EntryList/EntryList.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { useEntry } from '../../context/EntryContext';
+import './EntryList.css';
+
+export default function EntryList() {
+  const { entry } = useEntry();
+
+  return (
+    <div className="message">
+      {entry.map(({ name, message }) => (
+        <div key={message} className="each-message">
```

Comment:
I might add the index to the message to be more confident your key will be unique